utils/drift_detection.py

```diff
@@ -96,7 +96,7 @@ def detect_uncertainty_label_drift(dataset, train_loader, model):
     drifting_classes = dataset.drifting_classes
```

Comment:
instead of having two very similar functions we can make label_drift as additional keyword argument to detect_uncertainty_drift